Auth/AuthAPI.py

```diff
@@ -1,7 +1,36 @@
 # Score API here
-from flask import Blueprint
+from flask import Blueprint, request
 import sys
+import jwt
 from db import db
+
 sys.path.append("../")
 
+
 auth_api = Blueprint("auth", __name__)
+
+
+
+@auth_api.route("/register", methods=['POST'])
+def register():
+    if request.method == 'POST':
+        username = request.args.get('username')
+        password = request.args.get('passwordHash')
+        if(username == '') or (password == '') :
+            return jsonify({'status':'error','message':'username or password field is empty'})
```

Comment:
If you want to use the jsonify method from the Flask package. `from Flask import jsonify, ..., ... `